src/ui/handlers/autocomplete-ui-handler.ts

```diff
@@ -3,15 +3,19 @@ import { Button } from "#enums/buttons";
 import { OptionSelectUiHandler } from "./option-select-ui-handler";
 
 export class AutoCompleteUiHandler extends OptionSelectUiHandler {
-  private modalContainer: Phaser.GameObjects.Container;
+  private modalContainer: Phaser.GameObjects.Container | null;
```

Comment:
NIT: It was decided to prefer the use of `undefined` instead of null for consistency

```suggestion
  private modalContainer?: Phaser.GameObjects.Container;
```